src/kv/RocksDBStore.cc

```diff
@@ -363,6 +366,26 @@ void RocksDBStore::close()
     cct->get_perfcounters_collection()->remove(logger);
 }
 
+void RocksDBStore::get_statistics(Formatter *f) {
+  std::string stats;
+  bool status = db->GetProperty("rocksdb.stats", &stats);
+  if (status) {
+    f->open_object_section("bluestore rocksdb statistics");
+    std::size_t p1 = 0, p2 = 0;
+    while ((p1 = stats.find("\n", p2)) != std::string::npos) {
+      if (p1 != p2) {
+	std::string substr = stats.substr(p2, p1 - p2 + 1);
+	substr = "  " + substr;
+	f->write_raw_data(substr.data());
+      } else
+	f->write_raw_data("\n");
+      p2 = ++p1;
+    }
+    f->write_raw_data("\n");
+    f->close_section();
```

Comment:
I don't think we want to be doing nay write_raw_data or else we're generate invalid json and xml both.  If the stats are preformatted then we can do a single dump_string().